cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc

```diff
@@ -84,7 +84,11 @@ Here we detail the available versions:
 |
 
 | 4.3
-| This will force the query to use Neo4j Cypher 4.3. As this is the default version, it is not necessary to use this option explicitly.
+| This will force the query to use Neo4j Cypher 4.3.
+| 
+
+| 4.4
+| This will force the query to use Neo4j Cypher 4.4. As this is the default version, it is not necessary to use this option explicitly.
```

Comment:
```suggestion
| This will force the query to use Neo4j Cypher 4.4. 
As this is the default version, it is not necessary to use this option explicitly.
```

Reply:
Yes, in 5.0, I'll create a line for 5.0. Thanks, @davidoliverSP2!